test/Org.test.js

```diff
@@ -0,0 +1,173 @@
+const { expect } = require('chai');
+// const sinon = require('sinon');
+const nock = require('nock');
+require('chai').use(require('sinon-chai'));
+const Org = require('../src/models/Org');
+
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-undef */
+
+describe('Org model', () => {
```

Comment:
@michaelmurray6298 nice work using `nock` here! Please address the `it` statements as described above, thanks!